pkg/ai/iai.go

```diff
@@ -49,7 +51,7 @@ type AIProvider struct {
 	Name     string `mapstructure:"name"`
 	Model    string `mapstructure:"model"`
 	Password string `mapstructure:"password"`
-	BaseURL  string `mapstructure:"base_url"`
+	BaseURL  string `mapstructure:"baseurl"`
```

Comment:
sorry about this. I've messed up during cherry-pick and rebase on the other PR, now works as intended (`baseurl` in place of `base_url`)